hw/ip/entropy_src/dv/env/entropy_src_env.core

```diff
@@ -5,6 +5,9 @@ CAPI=2:
 name: "lowrisc:dv:entropy_src_env:0.1"
 description: "ENTROPY_SRC DV UVM environment"
 filesets:
+  files_rtl:
+    depend:
+        - lowrisc:ip:entropy_src:0.1
```

Comment:
this and line 41 seem not relevant and not needed

Reply:
Shouldn't have been any changes to an entropy_src file so I removed it from this PR